theme/static/css/hydroshare_core.css

```diff
@@ -1908,6 +1908,17 @@ a.tag:hover {
     width: 360px;
   }
 }
+@media (min-width: 1440px) {
```

Comment:
@devincowan This selector is too general and will cause unexpected layout errors. The `container` class is used in many components and not specific to the main container.

Take for example this padding that would now show up in the resource landing page.
![image](https://user-images.githubusercontent.com/2448568/162038220-9f1c95db-e65d-49cf-b127-1aecfd0f136b.png)

A more specific selector is needed that targets only the relevant container in the base template.

Reply:
thanks @Maurier good catch I didn't notice that HS has nested `.container`s. Fixed that.

I do think we want to target all of the containers to override their width, not just the `.main-container`.
Otherwise, rows like the [left-header](https://github.com/hydroshare/hydroshare/blob/bootstrap-play-w-width/theme/templates/resource-landing-page/resource-header.html#L3) that you pointed out wouldn't get the updated full width -- they would still get the default Bootstrap width.

Reply:
Right. The conflicting part is then the `max-width`.

Reply:
yep I removed that thanks! [revert for nested containers](https://github.com/hydroshare/hydroshare/pull/4558/commits/9905d419388d0996b035d985d0083c0c0cdfc334)